src/activities/Elsa.Activities.SQL/Activities/ExecuteSqlServerQuery.cs

```diff
@@ -0,0 +1,57 @@
+using Elsa.Services;
+using Elsa.Attributes;
+using Elsa.Services.Models;
+using Elsa.ActivityResults;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Activities.ExecuteSqlServerQuery.Client;
+using Elsa.Activities.ExecuteSqlServerQuery.Factory;
+
+namespace Elsa.Activities.ExecuteSqlServerQuery.Activities
+{
+    /// <summary>
+    /// Execute an SQL query on given database using connection string
+    /// </summary>
+    [Trigger(
+        Category = "Execute Sql Query",
+        Description = "Run SQL scripts",
+        Outcomes = new string[0]
+    )]
+    public class ExecuteSqlServerQuery : Activity
+    {
+        [ActivityInput(
+            Hint = "SQl script to execute",
+            UIHint = ActivityInputUIHints.MultiLine,
+            SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid }
+        )]
+        public string Query { get; set; } = default!;
+
+        /// <summary>
+        /// The ConnectionStrings to run SQL
```

Comment:
Should it be singular: "connection string" instead? Also notice the use of regular English: "connection string" rather than "ConnectionStrings". Same comment for the `Hint` property.